README.md

```diff
@@ -38,25 +38,25 @@ Find us in the [**#unreal** channel](https://discordapp.com/channels/31127363330
 Visit the **feedback** section in our [forums](https://forums.improbable.io/) and use the **unreal-gdk** tag. [This link](https://forums.improbable.io/new-topic?category=Feedback&tags=unreal-gdk) takes you there and pre-fills the category and tag.
 
 **GitHub issues**</br>
-Create an issue in [this repository](https://github.com/improbable/UnrealGDK/issues).
+Create an issue in [this repository](https://github.com/spatialos/UnrealGDK/issues).
 
 ## Contributions
 We are not currently accepting public contributions - see our [contributions](.github/CONTRIBUTING.md) policy. However, we are accepting issues and we do want your feedback.
 
 ## Documentation
 For guidance on using the SpatialOS Unreal GDK, see the [documentation in this GitHub repository](docs/readme.md).
 
-For guidance on SpatialOS concepts see the documentation on the [SpatialOS website](https://docs.improbable.io/reference/13.0/shared/concepts/spatialos). <br/> 
+For guidance on SpatialOS concepts see the documentation on the [SpatialOS website](https://docs.improbable.io/reference/13.1/shared/concepts/spatialos). <br/> 
 
 ## Support
 We are not supporting the pre-alpha release for game development, as this version is for evaluation only. However, please give us your [feedback](#give-us-feedback).
 
 ## Roadmap
 
-TODO add roadmap
+See the [latest roadmap (Trello board)](https://trello.com/b/7wtbtwmL/unreal-gdk-roadmap) and let us know what you'd like to see in there.
```

Comment:
Is this the proper link?

Reply:
it's the one Giray sent me this morning - he said "This is currently not visible by public but we can do that at release."